File: Code/Components/DanpheEMR.ServerModel/SystemAdminModels/SqlAuditModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DanpheEMR.ServerModel
{
    public class SqlAuditModel
    {
        public DateTime? Event_Time { get; set; }
        public string Server_Instance_Name { get; set; }
        public string Database_Name { get; set; }
        public string Statement { get; set; }
        public string Server_Principal_Name { get; set; }
        public string Action_Id { get; set; }
        public string Object_Name { get; set; }
        public Int16? Session_Id { get; set; }
        public string Schema_Name { get; set; }        

    }
}

File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/system-admin/shared/sql-audit.model.ts

import {
    NgForm,
    FormGroup,
    FormControl,
    Validators,
    FormBuilder,
    ReactiveFormsModule
} from '@angular/forms';
import * as moment from 'moment/moment';
export class SqlAuditModel {
    public DateTime: Date = null;
    public Server_Instance_Name: string = "";
    public Database_Name: string = "";
    public Statement: string = "";
    public Server_Principal_Name: string = "";
    public Action_Id: string = "";
    public Object_Name: string = "";
    public Session_Id: string = "";
    public Schema_Name: string = "";
}
